frame/staking/src/lib.rs

```diff
@@ -402,12 +429,28 @@ pub enum RewardDestination<AccountId> {
 	None,
 }
 
-impl<AccountId> Default for RewardDestination<AccountId> {
+impl<AccountId: Clone> Default for RewardDestination<AccountId> {
 	fn default() -> Self {
 		RewardDestination::Staked
 	}
 }
 
+impl<AccountId: Clone> RewardDestination<AccountId> {
+	fn to_payout_destination(
```

Comment:
I think this function can simply be substituted in favor of an `impl From<RewardDestination> for PayoutDestination`

Reply:
I opted for this impl function because we cannot derive the actual stash and controller `AccountId` from `RewardDestination`.